src/System.Data.SqlClient/src/System/Data/SqlClient/SqlDbColumn.cs

```diff
@@ -0,0 +1,95 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+using System.Data.Common;
+
+namespace System.Data.SqlClient
+{
+    internal class SqlDbColumn : DbColumn
+    {
+        private readonly bool _isBrowseModeInfoConsumed;
+        private readonly _SqlMetaData _metadata;
```

Comment:
I looked over this again and it occurred to me: Are these fields even necessary? They are used to populate the properties, but after that, aren't used.